modules/nw-kuryr-migration-about.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-kuryr-ovn-kubernetes-migration-about_{context}"]
+= Migration to the OVN-Kubernetes network provider
+
+Migrating to the OVN-Kubernetes network plug-in is a manual process that includes some downtime during which your cluster is unreachable. Note, the migration is a one-way process.
+
+A migration to the OVN-Kubernetes network plug-in is supported basically on {rh-openstack-first} based platform.
+
+[id="considerations-kuryr-migrating-network-provider_{context}"]
+== Considerations for migrating to the OVN-Kubernetes network provider
+
+The subnets assigned to nodes and the IP addresses assigned to individual pods are not preserved during the migration.
```

Comment:
I get the reason behind Pods IP not being preserved, but why the Subnets defined for the Nodes are not preserved?